src/components/auto-dialog.tsx

```diff
@@ -97,7 +97,7 @@ export function AutoDialog({open, onOpenChange, settings}: EditFieldDialogProps)
                         {error}
                     </Alert>}
 
-                    <DialogFooter className="sm:justify-end">
+                    <DialogFooter className="sm:justify-end pb-4">
```

Comment:
Why this change?

Reply:
added indentation for the button, it was pressed to the bottom of the window